src/CommunityToolkit.Maui/Converters/ICommunityToolkitValueConverter.shared.cs

```diff
@@ -5,6 +5,16 @@ namespace CommunityToolkit.Maui.Converters;
 /// <inheritdoc />
 public interface ICommunityToolkitValueConverter : IValueConverter
 {
+	/// <summary>
+	/// Type Converting From
+	/// </summary>
+	Type FromType { get; }
+
+	/// <summary>
+	/// Type Converting To
```

Comment:
Perhaps something like

```suggestion
	/// Gets the <see cref="Type" /> this converter expects to <see cref="Convert" /> to or <see cref="ConvertBack" /> from.
```